fagiServer/src/test/java/com/fagi/mockhelpers/ConversationMocks.java

```diff
@@ -3,15 +3,37 @@
 import com.fagi.conversation.Conversation;
 import com.fagi.conversation.ConversationType;
 import com.fagi.model.Data;
+import com.fagi.model.User;
+import com.fagi.model.messages.message.TextMessage;
+import org.apache.commons.text.RandomStringGenerator;
```

Comment:
Rather than use a whole library just write this yourself:
```
public class TestHelper {
    private final String SALT_CHARS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890";

    **
     * Create a random string 
     *
     * @param length the required length of the string
     * @return random string with the required length.
     * @throws IllegalArgumentException if the length is less then 1.
     */
    public String getSaltString(int length) {
        if (length < 1) {
            throw new IllegalArgumentException(length + ": is not a valid length. Must be larger than 1.")
        }

        StringBuilder salt = new StringBuilder();
        Random rnd = new Random();

        while (salt.length() < length) {
            int index = (int) (rnd.nextFloat() * SALTCHARS.length());
            salt.append(SALTCHARS.charAt(index));
        }
        return salt.toString();
    }
}
```